build/teamcity-verify-archive.sh

```diff
@@ -10,4 +10,4 @@ docker run \
   --rm \
   --volume="$(cd "$(dirname "$0")" && pwd):/work" \
   --workdir="/work" \
-  golang:1.8.1 ./verify-archive.sh
+  golang:stretch ./verify-archive.sh
```

Comment:
This will become Go 1.9 the day Go 1.9 is released. I guess that's okay, and maybe even desirable?